server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -1437,7 +1437,7 @@ public void setRemoteStateReadTimeout(TimeValue remoteStateReadTimeout) {
         this.remoteStateReadTimeout = remoteStateReadTimeout;
     }
 
-    private BlobStoreTransferService getBlobStoreTransferService() {
+    public BlobStoreTransferService getBlobStoreTransferService() {
```

Comment:
Is this being used in IT?

Reply:
Yes.

Reply:
Can you point me to the new usage due to which we need to change access?

Reply:
My bad, was using it earlier. Remove it.